hudi-utilities/src/main/java/org/apache/hudi/utilities/callback/kafka/HoodieWriteCommitKafkaCallbackUtil.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.utilities.callback.kafka;
+
+import org.apache.hudi.callback.util.HoodieWriteCommitCallbackUtil;
+import org.apache.hudi.common.util.StringUtils;
+import org.apache.hudi.common.util.ValidationUtils;
+
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+
+import java.util.Properties;
+
+import static org.apache.hudi.config.HoodieWriteCommitCallbackConfig.CALLBACK_KAFKA_ACKS;
+import static org.apache.hudi.config.HoodieWriteCommitCallbackConfig.CALLBACK_KAFKA_BOOTSTRAP_SERVERS;
+import static org.apache.hudi.config.HoodieWriteCommitCallbackConfig.CALLBACK_KAFKA_PARTITION;
+import static org.apache.hudi.config.HoodieWriteCommitCallbackConfig.CALLBACK_KAFKA_RETRIES;
+import static org.apache.hudi.config.HoodieWriteCommitCallbackConfig.CALLBACK_KAFKA_TOPIC;
+import static org.apache.hudi.config.HoodieWriteConfig.TABLE_NAME;
+
+/**
+ * Util helps to build kafka callback.
+ */
+public class HoodieWriteCommitKafkaCallbackUtil {
```

Comment:
IMO, moving the content of this class into `HoodieWriteCommitKafkaCallback ` is reasonable. WDYT?

Reply:
> 
> 
> IMO, moving the content of this class into `HoodieWriteCommitKafkaCallback ` is reasonable. WDYT?

I once hesitated here, kind of messy to host in one class. But it is reasonable, will move